# 4.2 The Normal Distribution and $z$-Scores

## Objectives ##
- Perform descriptive data analysis of collected data including calculation and interpretation of measures of central tendency, measures of variation, measures of position, detection of outliers, and applying the Empirical Rule to bell-shaped data.
- Apply probability and counting rules (including addition and multiplication rules) as pertaining to both continuous random variables and discrete random variables.
- Compute probabilities using the Normal and Binomial Distributions, compute the mean, variance, and standard deviation of Normal and Binomial distributions.
- Analyze an application in the disciplines business, social sciences, psychology, life sciences, health sciences, and education, and utilize the correct statistical process to arrive at a solution.

## Finding $z$-Scores
In the last section, we learned how to find probabilities for the standard normal distribution, $Z \sim N(0, 1)$. To find probabilities generally for a non-standard normal distribution, $X \sim N(\mu, \sigma)$, we first convert the $x$ value of interest into a **$z$-score**.

**A $z$-score is measured in units of the standard deviation.** For example, if the mean of a normal distribution is $\mu = 5$ and the standard deviation is $\sigma = 2$, the value 11 is three standard deviations above (or to the right of) the mean. The calculation is as follows:

$$x = \mu + z\sigma = 5 + (3)(2) = 11$$

So the $z$-score  of $x = 11$ is $z = 3$.

The mean for the standard normal distribution is zero, and the standard deviation is one. The transformation $z =  x−μσ$  produces the distribution $Z \sim N(0, 1)$. The value $x$ in the given equation comes from a normal distribution with mean $\mu$ and standard deviation $\sigma$. The probability (that is, area) to the left of $x$ in a normal distribution $X \sim N(\mu, \sigma)$ is equal to the probability to the left of the corresponding $z$ in the standard normal distribution $Z \sim N(0, 1)$.

If $X \sim N(\mu, \sigma)$, then the $z$-score is:

$$ z = \frac{x - \mu}{\sigma} $$

**The $z$-score tells you how many standard deviations the value $x$ is above (to the right of) or below (to the left of) the mean, $\mu$.** Values of $x$ that are larger than the mean have positive $z$-scores, and values of $x$ that are smaller than the mean have negative $z$-scores. If $x$ equals the mean, then $x$ has a $z$-score of zero.

***

### Example 2.1 ###

In [ ]:
#**VID=9u3Ja0rD70Y**#

***


### Example 2.2

The heights of adult men in the United States are normally distributed with a mean of 70 inches (5 ft, 10 in) and a standard deviation of 4 inches. If you choose an adult male from the U.S. population at random, what is the probability that:
1. he will be taller than 72 inches (12 ft)?
2. he will be shorter than 60 inches (5 ft)?
3. he will be between 60 inches and 72 inches tall?

#### Solution
The distribution of the heights of adult males is $X \sim N(70, 4)$. This is a non-standard normal distribution. To find the probabilities, we will convert to the standard normal distribution by finding the appropriate $z$-scores.

##### Part 1
We want $P(x > 72)$. To find the $z$-score associated with $x = 72$, we calculate

$$ z = \frac{x - \mu}{\sigma} = \frac{72 - 70}{4} = 0.5. $$

So $P(x > 72) = P(z > 0.5)$. We can now use R to find the probability:

In [1]:
pnorm(q = 0.5, lower.tail = FALSE)

[1] 0.3085375

So $P(x > 72) = P(z > 0.5) = 0.3085$. In other words, there is a 30.85% chance that the adult male you randomly choose from the population is more than 6 feet tall.

##### Part 2
We want $P(x < 60)$. To find the $z$-score associated with $x = 60$, we calculate

$$ z = \frac{x - \mu}{\sigma} = \frac{60 - 70}{4} = -2.5. $$

So $P(x < 60) = P(z < -2.5)$. Now use R to find the probability:

In [2]:
pnorm(q = -2.5, lower.tail = TRUE)

[1] 0.006209665

Then $P(x < 60) = P(z < -2.5) = 0.0062$. In other words, there is only a 0.62% chance that the adult male you randomly choose from the population is less than 5 feet tall.

##### Part 3
We want $P(60 < x < 72)$. From parts 1 and 2, we already know that the $z$-score associated with $x = 60$ is $z = -2.5$, and the $z$-score associated with $x = 72$ is $z = 0.5$. Then

$$ P(60 < x < 72) = P(-2.5 < z < 0.5) $$

We can use R to find this probability: we will find all the area to the left of $z = 0.5$, then subtract off the excess area to the left of $z = -2.5$.

In [3]:
pnorm(q = 0.5, lower.tail = TRUE) - pnorm(q = -2.5, lower.tail = TRUE)

[1] 0.6852528

So $P(60 < x < 72) = P(-2.5 < z < 0.5) = 0.6853$. That is, there is a 68.53% chance that the adult male you choose is between 5 feet and 6 feet tall.

***

### Example 2.3 ###

In [ ]:
#**VID=zCDeQAE8PSY**#

***

## Using $z$-Scores to Find Values
Sometimes we have a $z$-score and we want to find the $x$-value of a normal distribution $X \sim N(\mu, \sigma)$. By solving for $x$ in the formula above, we see that we can do this using the formula

$$ x = \mu + z\sigma $$

***

### Example 2.4
Scores on a calculus test are normally distributed with a mean score of 73 and a standard deviation of 12.
1. Find the 80th percentile.
2. 10% of the scores are higher than which score?

#### Solution
##### Part 1
To find the 80th percentile, start by finding the $z$-score.

In [4]:
qnorm(p = 0.80, lower.tail = TRUE)

[1] 0.8416212

So the $z$-score associated with the 80th percentile is $z = 0.8416$. To find the $x$-value, use the formula:

$$x = \mu + z\sigma = 73 + 0.8416(12) = 83.0992 $$

So a score of 83.0992 is higher than 80% of the scores.

##### Part 2
We want to find the $x$-value so that 10% of test scores are higher than that value. Start by finding the $z$-score.

In [5]:
qnorm(p = 0.10, lower.tail = FALSE)

[1] 1.281552

So 10% of the area of the standard normal distribution is to the right of $z = 1.2816$. Let's find the associated $x$-value:

$$ x = \mu + z\sigma = 73 + 1.2816(12) = 88.3792 $$

So only 10% of students scored higher than $x = 88.3792$.

***

### Example 2.5 ###

In [1]:
#**VID=GFLbgN6df0Q**#

***

## The Empirical Rule

Notice that the probability of a normally distributed random variable is determined by its $z$-scores. That is, we need only know how many standard deviations away from the mean a value is to calculate the probability.

This observation leads us to the **Empirical Rule**: If X is a random variable and has a normal distribution with mean µ and standard deviation σ, then the Empirical Rule states the following:

- 68% of the data values lie within one standard deviation of the mean (between $\mu - 1\sigma$ and $\mu + 1\sigma$).
- 95% of the data values lie within two standard deviations of the mean (between $\mu - 2\sigma$ and $\mu + 2\sigma$).
- 99.7% of the data values lie within three standard deviations of the mean (between $\mu - 3\sigma$ and $\mu + 3\sigma$).

<img src="empiricalrule.png" alt="Graphical breakdown of the Empirical Rule.">

### Example 2.6
IQ scores are normally distributed with a mean of 100 and a standard deviation of 15.
1. 68% of IQ scores lie between which two values?
2. 95% of IQ scores lie between which two values?
3. 99.7% of IQ scores lie between which two values?

#### Solution
##### Part 1
Since IQ scores are normally distributed, we know that 68% of IQ scores lie within one standard deviation of the mean. Let's find the values that are one standard deviation away from the mean:

$$ x = \mu + 1\sigma = 100 + 1(15) = 115 $$
$$ x = \mu - 1\sigma = 100 - 1(15) = 85 $$

So 68% of IQ scores are between 85 and 115.

##### Part 2
We know that 95% of IQ scores lie within two standard deviations of the mean. Let's find the values that are two standard deviations away from the  mean:

$$ x = \mu + 2\sigma = 100 + 2(15) = 130 $$
$$ x = \mu - 2\sigma = 100 - 2(15) =  70 $$

So 95% of IQ scores are between 70 and 130.

##### Part 3
We know that 99.7% of IQ scores lie within three standard deviations of the mean. Let's find the values that are three standard deviations away from the  mean:

$$ x = \mu + 3\sigma = 100 + 3(15) = 145 $$
$$ x = \mu - 3\sigma = 100 - 3(15) =  55 $$

So 99.7% of IQ scores are between 55 and 145.

*** 

### Example 2.7 ###

In [2]:
#**VID=1tTwrjPqIsk**#

***

## Attribution
Significant portions of this textbook were written by Barbara Illowsky and Susan Dean in their textbook *Introductory Statistics*, published by OpenStax.

Access for free at [https://openstax.org/books/introductory-statistics/pages/1-introduction](https://openstax.org/books/introductory-statistics/pages/1-introduction)

The image illustrating the Empirical Rule was taken from *Mathematics for the Liberal Arts* under a [CC BY: Attribution](https://creativecommons.org/licenses/by/4.0/) license.

Access for free at [https://courses.lumenlearning.com/math4libarts/chapter/the-empirical-rule/]